ban/tests/http/test_housenumber.py

```diff
@@ -430,3 +430,36 @@ def test_housenumber_select_use_default_orderby(get):
     assert resp.json['collection'][3]['ordinal'] == 'bis'
     assert resp.json['collection'][4]['number'] == '2'
     assert resp.json['collection'][4]['ordinal'] == 'ter'
+
+
+@authorize
+def test_cannot_duplicate_number_and_ordinal_for_same_parent(client):
+    assert not models.HouseNumber.select().count()
```

Comment:
Just a way to make sure the check line 446 is actually meaning what I think it means :)